common/utils/src/main/resources/error/README.md

```diff
@@ -116,1241 +116,4 @@ The following SQLSTATEs are collated from:
 - PostgreSQL 15
 - Oracle 12 (last published)
 - SQL Server
-- Redshift.
-
-<!-- SQLSTATE table start -->
-|SQLSTATE |Class|Condition                                         |Subclass|Subcondition                                                |Origin         |Standard|Used By                                                                     |
```

Comment:
This is not public doc, correct?

Reply:
Correct, not public. I will be opening separate PRs to update our public facing documentation on error classes.